content/blog/2016/2016-01-05-new-website.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "A new Jenkins website"
+tags:
+- www
+author: rtyler
+---
+
+When link:https://github.com/rtyler[I] first started working on the Jenkins
+website (then called by a different name) I selected
+link:https://drupal.org[Drupal], an extensible content management system, to
+get the job done. Like Jenkins itself, Drupal is rather easy to set up, install
```

Comment:
This sentence bounces awkwardly - "rather easy to setup, install plugins and authoring content is..."?